rakelib/form526.rake

```diff
@@ -219,31 +226,6 @@ namespace :form526 do
     end
   end
 
-  def create_submission_hash(claim_id, submission, user_uuid)
```

Comment:
these methods aren't called anywhere and were part of an old data migration task.